grakn-graql/src/main/java/ai/grakn/graql/internal/util/CommonUtil.java

```diff
@@ -48,6 +49,18 @@
 
     private CommonUtil() {}
 
+    public static <T> T withImplicitConceptsVisible(GraknGraph graph, Supplier<T> function) {
```

Comment:
That's true. Do you know a good place to put them?

I could also just move the whole of `CommonUtil` into `grakn-core`.